e2e-tests/micronaut-rest-api/src/test/java/org/evomaster/e2etests/micronaut/rest/TcpPortMicronautEMTest.java

```diff
@@ -0,0 +1,63 @@
+package org.evomaster.e2etests.micronaut.rest;
+
+import com.foo.micronaut.rest.MicronautTestController;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.utils.RestTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.core.Is.is;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TcpPortMicronautEMTest extends RestTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+        RestTestBase.initClass(new MicronautTestController());
+    }
+
+    @Test
+    public void testRunEM() throws Throwable {
+
+        runTestHandlingFlaky("TcpPortMicronautEMTest", "com.foo.TcpPortMicronautEMTest", 100, false, (args) -> {
```

Comment:
is there any reason why you set `createTests` false?

Reply:
Since other functions didn't work properly. I've set it to true